Userland/Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -210,7 +210,30 @@ void InfinitelyScrollableTableView::drop_event(GUI::DropEvent& event)
     m_is_dragging_for_cut = false;
     set_override_cursor(Gfx::StandardCursor::Arrow);
     auto drop_index = index_at_event_position(event.position());
-    AbstractTableView::set_cursor(drop_index, SelectionUpdate::Set);
+    if (selection().size() > 0) {
+        // Note: Get top left index position of previous selection
+        auto top_left_most_index = selection().first();
+        selection().for_each_index([&](auto& index) {
+            if (index.row() < top_left_most_index.row())
+                top_left_most_index = index;
+            else if (index.column() < top_left_most_index.column())
+                top_left_most_index = index;
+        });
+
+        // Note: Compare with drag location
+        auto x_diff = drop_index.column() - top_left_most_index.column();
+        auto y_diff = drop_index.row() - top_left_most_index.row();
+
+        // Note: Set new selection
```

Comment:
No need for "Note" here or in most other places you've added them.
Comments are already notes.